ports/esp32/boards/WAVESHARE_S3_ZERO/mpconfigboard.h

```diff
@@ -0,0 +1,13 @@
+#ifndef MICROPY_HW_BOARD_NAME
+// Can be set by mpconfigboard.cmake.
+#define MICROPY_HW_BOARD_NAME               "ESP32-S3-Zero module"
+#endif
+#define MICROPY_HW_MCU_NAME                 "ESP32S3"
+
+#define MICROPY_PY_MACHINE_DAC              (0)
+
+// Enable UART REPL for modules that have an external USB-UART and don't use native USB.
+#define MICROPY_HW_ENABLE_UART_REPL         (1)
+
```

Comment:
The UART pins should probably be defined as they are explicitly labelled:

![image](https://github.com/micropython/micropython/assets/194201/5c401b6e-3792-4c73-bf9f-0cbf4e4f324a)

Unfortunately, I'm not sure which pins they're connected to (is there a schematic anywhere? I couldn't find one!).

Reply:
usb the UART0 default pin,TX0 is GPIO43, RX0 is GPIO44